pkg/option/config_test.go

```diff
@@ -1372,19 +1372,7 @@ func TestDaemonConfig_StoreInFile(t *testing.T) {
 	Config.DryMode = true
 	err = Config.ValidateUnchanged(context.Background())
 	assert.Error(t, err)
-	strErr := strings.ReplaceAll(err.Error(), "\u00a0", " ")
-	assert.Equal(t, strErr, `Config differs:
-  &option.DaemonConfig{
-  	... // 1 ignored and 15 identical fields
-  	DatapathMode: "",
-  	RoutingMode:  "",
-- 	DryMode:      false,
-+ 	DryMode:      true,
-  	RestoreState: false,
-  	KeepConfig:   false,
-  	... // 5 ignored and 316 identical fields
-  }
-`)
+	assert.True(t, strings.Contains(err.Error(), "Config differs:"), "unexpected error output: %s", err.Error())
```

Comment:
assert already provides a corresponding function. IMO, there's also no need to output the error in the message, as this will already be handled by the assertion lib.

->

```
assert.ErrorContains(t, err, "Config differs:", "Should return a validation error")
```

Reply:
Done, thanks for the edit, always learn something new :-)